modules/integration/tests-integration/tests-backend/src/test/java/org/wso2/identity/integration/test/oauth2/OAuth2DeviceFlowTestCase.java

```diff
@@ -297,19 +299,32 @@ public HttpResponse sendGetRequest(HttpClient client, String locationURL) throws
     /**
      * Create Application with the given app configurations
      *
-     * @return OAuthConsumerAppDTO
-     * @throws Exception
+     * @return ApplicationResponseModel
+     * @throws Exception exception
      */
-    private OAuthConsumerAppDTO createApp() throws Exception {
-
-        OAuthConsumerAppDTO appDTO = new OAuthConsumerAppDTO();
-        appDTO.setApplicationName(OAuth2Constant.OAUTH_APPLICATION_NAME);
-        appDTO.setCallbackUrl(OAuth2Constant.CALLBACK_URL);
-        appDTO.setOAuthVersion(OAuth2Constant.OAUTH_VERSION_2);
-        appDTO.setGrantTypes("authorization_code implicit password client_credentials refresh_token " +
-                             "urn:ietf:params:oauth:grant-type:saml2-bearer iwa:ntlm " +
-                             "urn:ietf:params:oauth:grant-type:device_code");
-        appDTO.setBypassClientCredentials(true);
-        return createApplication(appDTO, SERVICE_PROVIDER_NAME);
+    private ApplicationResponseModel createApp() throws Exception {
+
+        ApplicationModel application = new ApplicationModel();
+
+        List<String> grantTypes = new ArrayList<>();
+        Collections.addAll(grantTypes, "authorization_code", "implicit", "password", "client_credentials", "refresh_token", "urn:ietf:params:oauth:grant-type:saml2-bearer", "iwa:ntlm", "urn:ietf:params:oauth:grant-type:device_code");
```

Comment:
Check the line length. Update IDE settings to limit the length to 120 characters.

Reply:
Fixed with https://github.com/wso2/product-is/pull/15906/commits/38e6cdd0e3f7357405e8fe1891714dcfcb17624b